app/models/concerns/weaknesses/score.rb

```diff
@@ -30,4 +30,27 @@ def take_as_old_for_score? date: Time.zone.today
       origination_date &&
       (origination_date < date - 2.years)
   end
+
+  def risk_weight
+    risk.next
+  end
+
+  def state_weight
+    being_implemented? ? 1 : 0
+  end
+
+  def age_weight date: Time.zone.today
+    days = date - follow_up_date
+
+    case
+    when days.to_i <= 2.years.to_i / 3600 / 24
+      1
+    when days.to_i <= 4.years.to_i / 3600 / 24
+      1.5
+    when days.to_i <= 6.years.to_i / 3600 / 24
+      2
+    else
+      2.5
+    end
+  end
```

Comment:
Lo ve bien a esto?

Reply:
Pero no hay necesidad, arriba estaba bien solo que le faltaba el .days a algunos.

Reply:
mmm. seguro la manquie en algun lugar pero no me funcionó. Si reviso 2.years.days = 63113904 days  (la unidad la pone solo, pero son segundos)

Reply:
Es que de un lado faltó el days, del otro sobró... te pasé varios ejemplos, funciona así:

```ruby
2.years < 400.days
```

No tenés que llevar todo a "days", sino a esos "números con unidades" y el resto ya funciona.

Nada como la documentación: https://guides.rubyonrails.org/active_support_core_extensions.html#extensions-to-numeric-time.